clio-client/src/main/scala/org/broadinstitute/clio/client/commands/MoveWgsUbamCommand.scala

```diff
@@ -96,7 +126,9 @@ object MoveWgsUbamCommand
     ioUtil.copyGoogleObject(source.get, destination.get) match {
       case 0 => Future.successful(())
       case _ =>
-        Future.failed(new Exception("Copy files in google cloud failed"))
+        logger.error(s"Copy files in google cloud failed from '${source
```

Comment:
The different error message texts refer to both "google cloud" and "cloud". Maybe just "cloud" would be better everywhere to be consistent? We are probably not working with other cloud storage systems for a while so saying "cloud" is unambiguous.